### ISSUE TITLE
CartoDB point layer not drawing with default markers

### ISSUE THREAD

**Issue by makella** [nationalparkservice/npmap-builder#190]
When I add a point layer from CartoDB, the points are drawing as circles with a black outline in Builder as opposed to the traditional default teardrop marker shape.

Example:
http://www.nps.gov/maps/embed.html?mapId=b16b1d22-2911-4a7b-8a0d-ea195bc7c81a

**Comment by nateirwin** [comment]
Yes, this is because we are passing CartoCSS to CartoDB for points, and it was easier to get circles w/outlines working initiallly. Maybe you can post a snippet of CartoCSS that will work to pull in a Maki marker, with a designated icon? I should be able to get this hooked up pretty easily once I have that in hand.

**Comment by makella** [comment]
I should have explained better... the points draw as the default point symbol (which I think is fine).  The main issue is that once added, the point symbol can't be changed to a teardrop marker with a symbol.  Does that make sense?  I thought previously after point data from a CartoDB table was added that we had the ability within Builder to modify it to a teardrop with a Maki/NPMaki icon. 

If by default we have a Maki marker drawing, will that be able to get symbolized like point data say from a GeoJSON?

![screen shot 2015-02-24 at 9 07 03 am](https://cloud.githubusercontent.com/assets/1566273/6353140/bb86ae86-bc04-11e4-859e-b2b7ff20365b.png)

**Comment by nateirwin** [comment]
Builder never supported adding markers to CartoDB layers; it has always only supported symbolized circles. I think we should probably default it to the same options that are available for GeoJSON overlays, but we'll need to figure out what CartoCSS needs to be passed to CartoDB.

Once I get this from you, I can update the interface for both CartoDB overlays and GeoJSON overlays, and support specifying either circle markers or Maki/NPMaki-based markers.

**Comment by makella** [comment]
@nateirwin this came up with @tpcolson today which reminded me it is in my court! I have a question about the teardrop markers... do we need CartoCSS for those in addition to which Maki/NPMaki icon should draw? Let's chat about this for a few so I understand exactly what you need.

**Comment by makella** [comment]
Workaround until implemented outlined here: https://github.com/nationalparkservice/npmap-builder/issues/209#issuecomment-113217846

**Comment by nateirwin** [comment]
This may be as easy as adding a checkbox for bringing a CartoDB overlay in as GeoJSON rather than as rasters. This would be similar to how it's implemented by the Places preset.
